tests-cuda/test_1300_same_for_numba_cuda.py

```diff
@@ -0,0 +1,1382 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
```

Comment:
the original PR number in the file name is kept because these tests are identical to the Numba ones in the `tests` directory, but the tests here run on a GPU using Numba CUDA.